run_sysbench.sh

```diff
@@ -0,0 +1,30 @@
+pgip=127.0.0.1
```

Comment:
Can we make this configurable? Because this is something a lot of users may need to change.

Reply:
Done.